app/components/Layout/Footer.jsx

```diff
@@ -25,6 +29,39 @@ class Footer extends React.Component {
         router: React.PropTypes.object
     };
 
+    constructor(props){
+        super(props);
+
+        this.state = {};
+
+        this.checkNewVersionAvailable.call(this);
```

Comment:
You shouldn't call async methods from the constructor, move them to componentDidMount instead. In this case it's also a little problematic to have all this code in the footer, since the footer isn't rendered when you're on the exchange page.